solvcon/parcel/gas/inout.py

```diff
@@ -65,12 +65,28 @@ def __init__(self, cse, show_bclist=False, perffn=None, **kw):
         super(MeshInfoHook, self).__init__(cse, **kw)
 
     def preloop(self):
+        from ...boundcond import BC
```

Comment:
`BC` is available in the `solvcon` top-level namespace.  This line is not necessary.